imap/jmap_backup.c

```diff
@@ -1143,15 +1143,13 @@ static int recreate_calendar(const mbentry_t *mbentry,
     if (!r && !*newmailbox) {
         /* Create the calendar */
         char *newmboxname = caldav_mboxname(req->accountid, makeuuid());
-        struct mboxlock *namespacelock = user_namespacelock(req->accountid);
```

Comment:
Don't we still want a namelock when recreating the mailbox?

Reply:
This is locked in the caller, so there's no need to lock it again.